charts/dave/README.md

```diff
@@ -0,0 +1,5 @@
+# dave-helm
```

Comment:
this README.md will be presented on ArtifactHub etc and should contain some documentation - see existing chart READMEs for good examples